README.md

```diff
@@ -4,6 +4,10 @@
 
 Developing a common ontology for energy system analysis and scenarios.
 
+## Scope of this ontology
+
+This **domain** ontology is a joint effort to represent the energy-system modelling context based on standard terminologies used by human experts in this field of research. It is designed to improve transparency and facilitate comparability & transferability within the energy system modelling context.
```

Comment:
@stap-m - Now you can click on "Add suggestion to batch" for this one and the next one, then "Commit suggestions (3)" from the last suggestion, this will commit them as one. Or you can commit them individually :)